src/macrofier/engine.mjs

```diff
@@ -338,18 +342,19 @@ const generateMacros = (obj, templates, languages, options = {}) => {
   const events = eventsArray.length ? getTemplate('/sections/events', templates).replace(/\$\{event.list\}/g, eventsArray.map(m => m.body).join('\n')) : ''
   const eventList = eventsArray.map(m => makeEventName(m))
   const defaults = generateDefaults(obj, templates)
-  const schemasArray = generateSchemas(obj, templates, { baseUrl: '', section: 'schemas' }).filter(s => (options.copySchemasIntoModules || !s.uri))
+  const schemasArray = generateSchemas(obj, templates, { baseUrl: '', section: 'schemas' }).filter(s => (options.copySchemasIntoModules || !s.uri)).reverse()
```

Comment:
Okay.

I'm fine if we keep this logic in C, but at some point, it might not work there either, since there's no guarantee that OpenRPC schema include schemas in any particular order.

Reply:
This dependency already removed, now schemas sorting based on ReferencedBySchema

Reply:
awesome!